airbyte-integrations/connectors/source-mongodb-v2/gradle.properties

```diff
@@ -0,0 +1 @@
+JunitMethodExecutionTimeout=2 m
```

Comment:
something was failing with default I guess?

Reply:
Yeah the default of 1m wasn't enough. See the notes in the PR description for when the regression was introduced. TLDR at some point the defaults went from 5m -> 1m in the cdk w/out testing on other connectors